zul/src/org/zkoss/zul/Textbox.java

```diff
@@ -220,6 +236,15 @@ public Boolean getValue(Component cmp) {
 				return ((Textbox) cmp).isMultiline();
 			}
 		});
+		_properties.put("submitByEnter", new BooleanPropertyAccess() {
```

Comment:
Why set  _properties.put(....), reason? is there  PropertyAccess pa = _properties.get(prop);??